docs/troubleshooting-guide/db-performance.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Troubleshooting database performance
+---
+
+# Troubleshooting database performance
+
+This guide deals with databases or data warehouses that are [connected to Metabase](../databases/connecting.md) as data sources.
+
+To fix problems with your Metabase [application database](../installation-and-operation/configuring-application-database.md), check out these troubleshooting guides:
+
+- [Running Metabase](./running.md).
+- [Running Metabase on Docker](./docker.md).
+- [Using or migrating from an H2 application database](./loading-from-h2.md)
+
+## Identifying bottlenecks
+
+1. Optional: Use Metabase's [auditing tools](../usage-and-performance-tools/audit.md) to look at your Metabase usage stats.\*
+2. Go to your database's server logs and check whether:
+   - Your tables are growing in size,
+   - More people are using Metabase to access your database,
+   - People are accessing your database more often, or
+   - Another script or application is accessing the database frequently.
+3. If specific tables are being queried a lot, try [Optimizing your table schemas](https://www.metabase.com/learn/analytics/which-data-warehouse).
+4. If your data, user base, and usage are starting to outgrow your database's resources, think about [scaling Metabase](https://www.metabase.com/learn/administration/metabase-at-scale) or [upgrading your hardware](https://www.metabase.com/learn/analytics/which-data-warehouse).
```

Comment:
This bullet seems imprecise. How would someone distinguish the culprit among their Metabase server, their Metabase application server, and their data warehouse?